app/assets/stylesheets/custom.css.scss

```diff
@@ -249,3 +245,87 @@ td {
     color:white;
     }
 }
+
+/* admin dashboard */
+
+.page-header {
+  border-bottom: 2px solid #5F8295;
+}
+
+.panel {
+  box-shadow: 0 1px 2px #C3C3C3;
+  color: #8B91A0;
+  margin-top: 20px;
+  background-color: #fff;
+  border: 1px solid transparent;
+  box-shadow: 0 1px 1px rgba(0,0,0,.05);
+}
+
+.panel-default {
+  border-color: #ddd;
+}
+
+.panel-white {
+  background-color: #ffffff;
+  position: relative;
+  color: #8B91A0;
+}
+
+.core-box .core-icon {
+  width: 33.3333%;
+  left: 0;
+  top: 0;
+  bottom: 0;
+  position: absolute;
+}
+
+.text-center {
+  text-align: center;
+}
+
+.core-box .core-content {
+  margin-left: 33.3333%;
+  min-height: 120px;
+  h2, h4 {
+    color: #4C4A4A;
+  }
+}
+
+.padding-10 {

```

Comment:
it's bad to use separate class for such things as padding
why you didn't add this to partition- class